morpholopy/galaxy_plots.py

```diff
@@ -657,63 +614,22 @@ def plot_galaxy(
     mass_map_face.convert_to_units("Msun / pc**2")
     mass_map_edge.convert_to_units("Msun / pc**2")
     mass_map_face_plot_HI = mass_map_face  # save for H2 ratio plot
-    totalmass_H2 = totalmass  # save for H2 ratio plot
+    totalmass_HI = totalmass  # save for H2 ratio plot
```

Comment:
The comment does not match the variable.

Reply:
The updated version was correct, the HI mass was used for the H2/HI ratio plot but this is already done elsewhere, and it is not used anymore. Deleted.